src/components/sync-sites-modal-selector.tsx

```diff
@@ -224,7 +235,30 @@ function SiteItem( {
 			} }
 		>
 			<div className="flex flex-col gap-0.5 min-w-0">
-				<div className={ cx( 'a8c-body truncate', ! isSyncable && 'text-a8c-gray-30' ) }>
+				<div
+					className={ cx(
+						'a8c-body truncate flex items-center',
+						! isSyncable && 'text-a8c-gray-30'
+					) }
+				>
+					{ isPressable && (
+						<span className="me-1.5">
+							<PressableLogo size={ 12 } />
+						</span>
+					) }
+					{ ! isPressable && (
+						<span className="me-1.5">
+							<WordPressLogoCircle
+								size={ 12 }
+								{ ...( isSelected && { color: '#fff' } ) }
+								{ ...( ( isUnsupported ||
+									needsUpgrade ||
+									isDeleted ||
+									isMissingPermissions ||
+									isNeedsTransfer ) && { color: '#8c8f94' } ) }
```

Comment:
It would be interesting to move this logic to a single variable/prop or inside the comopnent.

Reply:
Good idea! I am addressing this in a separate PR now: https://github.com/Automattic/studio/pull/1416.